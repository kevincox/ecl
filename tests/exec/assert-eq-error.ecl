assert 1 == error: "test"
