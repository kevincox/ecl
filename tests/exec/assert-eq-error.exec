cmd: load assert-eq-error.ecl
status: 1
