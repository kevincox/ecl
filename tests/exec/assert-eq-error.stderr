At line 1.10 On right side of ==
Error: "test"

Error occurred.
